dlc/src/test/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelfTest.scala

```diff
@@ -0,0 +1,118 @@
+package org.bitcoins.dlc
+
+import org.bitcoins.core.config.RegTest
+import org.bitcoins.core.crypto.{
+  DoubleSha256DigestBE,
+  ECPrivateKey,
+  Schnorr,
+  SchnorrNonce
+}
+import org.bitcoins.core.currency.{CurrencyUnits, Satoshis}
+import org.bitcoins.core.number.UInt32
+import org.bitcoins.core.protocol.script.P2PKHScriptPubKey
+import org.bitcoins.core.protocol.transaction.TransactionOutPoint
+import org.bitcoins.core.script.crypto.HashType
+import org.bitcoins.core.util.{BitcoinScriptUtil, CryptoUtil}
+import org.bitcoins.core.wallet.fee.SatoshisPerByte
+import org.bitcoins.core.wallet.utxo.P2PKHSpendingInfo
+import org.bitcoins.testkit.util.BitcoinSAsyncTest
+import scodec.bits.ByteVector
+
+import scala.concurrent.Future
+import scala.concurrent.duration.DurationInt
+
+class BinaryOutcomeDLCWithSelfTest extends BitcoinSAsyncTest {
+  behavior of "BinaryOutcomeDLCWithSelf"
+
+  it should "work" in {
```

Comment:
so shouldn't there be test cases for 

1. Local wins, cooperative close
2. Remote wins, cooperative close
3. Local wins, non-cooperative close
4. Remote wins, non cooperative close
5. Anything I am missing?

Reply:
I am not adding support for cooperative close on this PR.

But I did add test cases so that we now cover all local and remote wins and loses cases.